packages/keystone/fetch.js

```diff
@@ -82,7 +82,7 @@ async function fetchWithLogger (url, options, extraAttrs) {
         startReqId,
         taskId: parentTaskId,
         execId: parentExecId,
-        headers: options.headers,
+        headers: JSON.stringify(options.headers),
```

Comment:
Sometimes it’s useful for fillers. May be we can add it to index?

Reply:
ok